features/git-kill/current_branch/on_feature_branch/without_remote_origin/with_open_changes.feature

```diff
@@ -0,0 +1,57 @@
+Feature: git kill: killing the current feature branch without a tracking branch (with open changes and without remote repo)
+
+  (see ../without_tracking_branch/with_open_changes.feature)
+
+
+  Background:
+    Given I have a feature branch named "other-feature"
+    And my repo does not have a remote origin
+    And I have a local feature branch named "current-feature"
+    And the following commits exist in my repository
+      | BRANCH          | LOCATION | MESSAGE                | FILE NAME            | FILE CONTENT            |
+      | other-feature   | local    | other feature commit   | other_feature_file   | other feature content   |
+      | current-feature | local    | current feature commit | current_feature_file | current feature content |
+    And I am on the "current-feature" branch
+    And I have an uncommitted file with name: "uncommitted" and content: "stuff"
+    When I run `git kill`
+
+
+  Scenario: result
+    Then it runs the Git commands
+      | BRANCH          | COMMAND                                |
+      | current-feature | git add -A                             |
+      |                 | git commit -m 'WIP on current-feature' |
+      |                 | git checkout main                      |
+      | main            | git branch -D current-feature          |
+    And I end up on the "main" branch
+    And the existing branches are
+      | REPOSITORY | BRANCHES            |
+      | local      | main, other-feature |
+    And now I have the following commits
+      | BRANCH        | LOCATION | MESSAGE              | FILE NAME          |
+      | other-feature | local    | other feature commit | other_feature_file |
+    And now I have the following committed files
```

Comment:
Since you have a very direct commit situation here (no merge commits, no repeated updates to the same files), the file content defined in the commit table matches the one on the file system, and you don't need an extra table to specify the file contents separately. 

```cucumber
And now I have the following commits
  | BRANCH        | LOCATION | MESSAGE              | FILE NAME          | FILE CONTENT          |
  | other-feature | local    | other feature commit | other_feature_file | other feature content |
```